mappings/net/minecraft/server/world/ChunkWatcher.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS tz net/minecraft/server/world/ChunkWatcher
```

Comment:
The name ChunkWatcher is a bit confusing if you don't know what the class is for in the first place (and I seem to recall that name from somewhere... in a slightly different context).

My proposed names: 
* `ChunkProviderEntry` - shows at least where it's used and isn't that confusing, but also doesn't explain a lot
 * `TrackedChunk` - less confusing, but may implicitly imply that it represents the chunk itself
 * `ChunkTrackingEntry` - clarifies that it's not the chunk itself, but it's also a bit verbose

Reply:
TrackedChunk is bad, I just renamed TrackedEntity to EntityTrackerEntry for the same reason.

Reply:
With the `ChunkManager` rename, `ChunkManagerEntry` also looks like a good name